lang/c++/api/Validator.hh

```diff
@@ -142,7 +142,7 @@ private:
     };
 
     std::vector<CompoundType> compoundStack_;
-    std::vector<size_t> counters_;
+    std::vector<int64_t> counters_;
```

Comment:
Since the counters cannot be negative, should this be `unit64_t`?

Reply:
updated